components/AvalancheForecastZoneMap.tsx

```diff
@@ -131,7 +262,7 @@ const AvalancheForecastZoneCard: React.FunctionComponent<{
   const dangerColor = colorFor(zone.danger_level);
 
   return (
-    <TouchableOpacity
+    <TouchableWithoutFeedback
```

Comment:
it's very distracting to have the opacity shift while dragging the drawer up/down. we'll have to see if we need an alternate form of feedback.